test/test_loadbalancing.yml

```
@@ -0,0 +1,38 @@
+  loadbalance-hash-1:

```

Missing `services:` line?

```suggestion
services:
  loadbalance-hash-1:
```

---

It's missing in more tests. For example: [test_keepalive.yml](https://github.com/rhansen/nginx-proxy/blob/keepalive/test/test_keepalive.yml) and [test_log_format.yml](https://github.com/nginx-proxy/nginx-proxy/blob/main/test/test_log_format.yml) and more.
So I was not sure what is prefered.

---

I should fix those.  Omitting it means use the old v1 semantics, which doesn't matter for those tests, but they should be updated anyway for consistency.  It is needed here in order to use `deploy.replicas`.